modules/rostests/apitests/crt/__dtoi64.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * PROJECT:         ReactOS API tests
+ * LICENSE:         GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:         Tests for __dtoi64
+ * COPYRIGHT:       Copyright 2021 Roman Masanin <36927roma@gmail.com>
+ */
+
+#include <apitest.h>
+
+typedef struct _DTOI64_TEST_DATA
+{
+    union
+    {
+        double double_value;
+        unsigned long long double_raw;
+    } test_data;
+    long long expected_longlong;
```

Comment:
maybe the name can be shortened (same for the other test too):
```suggestion
    long long expected;
```